packages/cosmwasm/src/lcdapi/wasm.spec.ts

```diff
@@ -430,6 +430,56 @@ describe("wasm", () => {
       expect(await client.wasm.getContractInfo(nonExistentAddress)).toBeNull();
     });
 
+    it("can list contract history", async () => {
+      pendingWithoutWasmd();
+      const wallet = await Secp256k1Wallet.fromMnemonic(alice.mnemonic);
+      const client = makeWasmClient(wasmd.endpoint);
+      const beneficiaryAddress = makeRandomAddress();
+      const transferAmount: readonly Coin[] = [
+        {
+          amount: "707707",
+          denom: "ucosm",
+        },
+      ];
+
+      // reuse an existing contract, or upload if needed
+      let codeId: number;
+      const existingInfos = await client.wasm.listCodeInfo();
+      if (existingInfos.length > 0) {
+        codeId = existingInfos[existingInfos.length - 1].id;
+      } else {
+        const uploadResult = await uploadContract(client, wallet, getHackatom());
+        expect(uploadResult.code).toBeFalsy();
+        const uploadLogs = parseLogs(uploadResult.logs);
+        const codeIdAttr = findAttribute(uploadLogs, "message", "code_id");
+        codeId = Number.parseInt(codeIdAttr.value, 10);
+      }
+
+      // create new instance and compare before and after
+      const result = await instantiateContract(client, wallet, codeId, beneficiaryAddress, transferAmount);
+      expect(result.code).toBeFalsy();
+      const logs = parseLogs(result.logs);
+      const contractAddressAttr = findAttribute(logs, "message", "contract_address");
+      const myAddress = contractAddressAttr.value;
+
+      // check out history
+      const myHistory = await client.wasm.getContractCodeHistory(myAddress);
+      assert(myHistory);
+      expect(myHistory).toEqual(
```

Comment:
To check the existence of an array element, you need `expect(myHistory).toContain(searchItem)`. If you only want to check some properties of the searchItem, you can combine it with `objectContaining`, e.g.

```ts
      expect(myHistory).toContain(
        jasmine.objectContaining({
          codeId: codeId,
          operation: "Init",
        }),
      );
```